src/Model/Request/Request.php

```diff
@@ -201,6 +204,11 @@ public function uri(): UriInterface
         return $this->uri;
     }
 
+    public function customQueryParameters(): KeyValueCollectionInterface
```

Comment:
This change must also take place in the `RequestInterface` and therefor we need to do a new major release of all three json-api packages. Please also extend the [documentation](https://github.com/dogado-group/json-api-common/blob/main/docs/07-requests.md) accordingly.